### PR TITLE
Smart imdb plugin

### PR DESCRIPTION
**Pull request by liiight**
This plugin enables generating entries based on an entity, an entity being a person, character or company. It's based on IMDBpy which is required (pip install imdbpy).

### CODE REVIEW

File: flexget/plugins/input/smart_imdb.py [hunk 1]
```diff
@@ -0,0 +1,478 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+import re
+from jsonschema.compat import str_types
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+from imdb import IMDb
```
Comment (cvium) [inline flexget/plugins/input/smart_imdb.py]: This will throw an exception if imdbpy is not installed? Should be handled with a try-catch block imo.

Comment (liiight) [inline flexget/plugins/input/smart_imdb.py]: I updated based on this feedback, but now i'm thinking i should have done the whole thing different, and split this into two plugins. one for the input and one for the filter... 
what do you think?

Comment (cvium) [inline flexget/plugins/input/smart_imdb.py]: No idea. I'm not quite sure what the plugin does internally and I don't have time to taker a more thorough look at it :stuck_out_tongue:

Comment (liiight) [inline flexget/plugins/input/smart_imdb.py]: ok then, then i'll guess i'll go with hunch and split it then :)

**Pull request closed without merging**